doc/usage.rst

```diff
@@ -177,6 +177,12 @@ The labgrid library provides two ways to configure targets with resources and
 drivers: either create the :any:`Target` directly or use :any:`Environment` to
 load a configuration file.
 
+.. warning::
+   After your script/application is done using labgrid as a library, ensure to
+   call the ``cleanup()`` function all created targets. This will deactivate all
```

Comment:
Maybe it would be better to use the [atexit module](https://docs.python.org/3/library/atexit.html) in the Target to trigger this automatically? We'd need to make sure to handle any exceptions gracefully.

Reply:
Neat, I'll add that for the target. Handling this should IMO be done by catching the exception, mentioning that an error happened and directing the user to call `cleanup()` himself to get the full exception and investigate the problem.